### ISSUE TITLE
Editing and dual-function button.

### ISSUE THREAD

**Issue by meteos77** [textbrowser/biblioteq#387]
Can you please add a dual-function button when editing a book: a “Save and Close” button.

**Comment by textbrowser** [comment]
You have a strange obsession with this.

**Comment by meteos77** [comment]
I use BiblioteQ to have maximum pleasure and I try to make requests to make it easier to use. 

From your point of view, it's an stange obsession to improve the interface, but trying to modify 50 or 60 books ...

It's true that it's not a revolutionary request !

**Comment by textbrowser** [comment]
If you need to modify so many entries after an import, something else has to change. Or after another process. Why are you modifying 50 books? Did the books change? Nope. Some status. Or some import. You're looking at it from the wrong perspective.

**Comment by textbrowser** [comment]
But adding a new button is not what I want.

**Comment by meteos77** [comment]
My ultimate goal is to have a newsletter (1 per month) containing :
- title.
- author.
- publisher.
- physical description (lccontrolnumber).
- isbn13.
- callnumber.
- Literary summary (description).
- I would like to have the ark identifier in the BiblioteQ database (alternate_id_1) (future).

exemple :
[newletter_ALBUMS JEUNESSE.pdf](https://github.com/user-attachments/files/18098265/newletter_ALBUMS.JEUNESSE.pdf)

**Comment by meteos77** [comment]
My approach :
I have 3 book sources and 2 file types
    - delivery of the central library. (provided an unimarc .pan file)
    - donations & purchases. (unimarc .pan file retrieved from a website)

- custom script to transform unimarc into csv compatible with BiblioteQ model-1.

- use biblioteq script/Scripts/biblioteq-download-images-sqlite.sh

- fill in missing information by hand (my request)
        - i started a python script to complete fields but problem with xml parser. (I don't have your programming skills).

- generation of the news file.

**Comment by meteos77** [comment]
With unimarc files 
The shortcomings are often the same
- ark identifier
- literary summary
- physical description.

In amazon images, the script does a good job except for isbn13 in 979xxxxxxxx.

**Comment by meteos77** [comment]
[Source_Achats..csv](https://github.com/user-attachments/files/18098403/Source_Achats.csv)
[Source_Bibliothèque_Centrale.pan.csv](https://github.com/user-attachments/files/18098405/Source_Bibliotheque_Centrale.pan.csv)

**Comment by textbrowser** [comment]
![ok](https://github.com/user-attachments/assets/843b2e38-5073-4580-9f2c-c049d5044d48)

**Comment by textbrowser** [comment]
I don't have time to parse all comments. Clarity in what's to be done or what is requested which has nothing to do with this ticket all lost to me because I have other things in life. Again, my time is my time. Put in requests with details in a single comment and related documents and stuff that helps me help you and I don't spend my entire life parsing comments. Or else you don't want help as that's how I'd interpret it. The amount of effort you put is functional on the results. If you don't put effort but expect me to parse everything, review everything, that's just nonsense.

**Comment by meteos77** [comment]
I apologize for the flow of explanations. I added ideas as I went along instead of preparing a single post, which would have helped clarity.


Thank you for your thoughts, I'm going to change my point of view and work on a “pre-import” script to have a cvs file as complete as possible to avoid having to complete after import by the GUI.

Having a post-import script is great for images (or other things to think about).
